Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertFaceWall.cs

```diff
@@ -31,9 +31,13 @@ public ApplicationObject FaceWallToNative(RevitFaceWall speckleWall)
       return appObj;
     }
 
-    if (speckleWall.surface == null)
+    if (
```

Comment:
Not sure, but wondering if it can be simplified?
if(speckleWall?.brep?.Surfaces?.FirstOrDefault() == null)

Just a thought, no sure the Count is also needed or if the early ? termination will evaluated to null == null (but I suspect i does?)

Reply:
Sure but the ?. isn't it giving you the same in tis case?

Reply:
Yeah you're right, i'll push a change asap

Reply:
Resolving, just pushed the change.